app/lib/features/news/pages/add_news_page.dart

```diff
@@ -103,9 +103,22 @@ class AddNewsState extends ConsumerState<AddNewsPage> {
       actions: selectedNewsPost == null
           ? []
           : [
-              IconButton(
+              OutlinedButton(
                 onPressed: () => selectActionItemDialog(context),
-                icon: const Icon(Atlas.plus_circle),
+                style: OutlinedButton.styleFrom(
+                  padding: const EdgeInsets.all(8),
+                  shape: RoundedRectangleBorder(
+                    borderRadius: BorderRadius.circular(16),
+                  ),
+                ),
+                child: Row(
+                  children: [
+                    const Icon(Icons.add),
+                    const SizedBox(width: 4),
+                    Text(L10n.of(context).action),
+                    const SizedBox(width: 4),
+                  ],
+                ),
```

Comment:
why not `OutlinedButton.icon(icon: const Icon(Icons.add), label : Text(L10n.of(context).action), ...)`?

ref https://api.flutter.dev/flutter/material/OutlinedButton/OutlinedButton.icon.html

Reply:
My Bad, let me do this change before proceed for merge.